Installation/changes.html

```diff
@@ -144,8 +161,83 @@ <h2 id="release4.11">Release 4.11 </h2>
 <!-- Cell Complexes and Polyhedra -->
 <!-- Arrangements -->
 <!-- Triangulations and Delaunay Triangulations -->
+  <h3> 2D Alpha Shapes </h3>
+  <ul>
+    <li><b>Breaking change</b>:
+      Mirrored the concepts of the 2D alpha shape package with those of the
+      3D Alpha Shapes package. Consequently, a new concept, <code>WeightedAlphaShapeTraits_2</code>,
+      is introduced to provide requirements on the traits class for 2D weighted
+      alpha shapes. All models of the concept <code>Kernel</code> are models of
+      this new concept.
+    </li>
+    <li>
+      The concept <code>AlphaShapeTraits_2</code> now provides requirements on
+      the traits class for 2D basic alpha shapes, and refines <code>DelaunayTriangulationTraits_2</code>.
+    </li>
+  </ul>
+  <h3> 2D and 3D Triangulations </h3>
+  <ul>
+    <li><b>Breaking change</b>:
+      Added a new functor requirement, <code>Construct_point_2</code>, to the concepts
+      <code>TriangulationTraits_2</code> and <code>RegularTriangulationTraits_2</code>
+      and a new functor requirement, <code>Construct_point_3</code>, to the concepts
+      <code>TriangulationTraits_3</code> and <code>RegularTriangulationTraits_3</code>.
+      All models of the concept <code>Kernel</code> already provide these functors.
+    </li>
+    <li><b>Breaking change</b>:
+      Introduced the concepts <code>RegularTriangulationVertexBase_2</code> and
+      <code>RegularTriangulationVertexBase_3</code>. These concepts describe
+      the requirements on classes meant to represent a vertex of a regular triangulation.
+      Concepts that previously refined <code>TriangulationVertexBase_2</code> or
+      <code>TriangulationVertexBase_3</code> but described in fact a vertex class
+      used in a regular triangulation, such as the concept <code>MeshVertexBase_3</code> in the 3D mesh
+      generation package, now refine the corresponding new regular vertex concept.
+    <li><b>Breaking change</b>:
+      Uniformized the point type across all vertex and cell concepts. The triangulation
+      point type name is now always <code>Point</code>. Note that this does not change
+      the requirements but only the name: <code>Point</code> is stil expected to be equal to
+      <code>Traits::Point_[23]</code> for basic and Delaunay triangulations or to
+      <code>Traits::Weighted_point_[23]</code> for regular triangulations. Consequently:
```

Comment:
I guess this answers my question about typedefs in this package review. However I'm still wondering if it's the best option